src/recruitment/recruitmentDataReducer.js

```diff
@@ -17,6 +18,11 @@ export default function recruitmentDataReducer(state = initialState, action) {
                 eierNavident: action.navIdent,
                 eierNavn: action.displayName,
             };
+        case SET_COMMENT:
+            return {
+                ...state,
+                notat: action.comment,
```

Comment:
Hvilket domeneord bruker vi for kommentarfeltet/notatfeltet? Burde vi bli enige om ett av dem og være konsekvent med bruken?

Reply:
Det står notatfelt og notater i frontend. Men har vært comment i pam-ad og frontend koden. Jeg kan endre til notat.